src/app/components/LiveLabel/index.styles.ts

```diff
@@ -1,13 +1,65 @@
-import { css, Theme } from '@emotion/react';
+import { css, Theme, keyframes } from '@emotion/react';
+
+const pulseAnimation = keyframes({
+  '0%': {
+    transform: 'scale(1)',
+    opacity: 1,
+  },
+  '50%': {
+    transform: 'scale(1)',
+    opacity: 0,
+  },
+  '100%': {
+    transform: 'scale(1)',
+    opacity: 1,
+  },
+});
 
 const styles = {
-  liveLabel: ({ palette, spacings, fontVariants }: Theme) =>
+  liveLabelText: ({ palette, spacings, fontVariants }: Theme) =>
     css({
       color: palette.LIVE_DARK,
       display: 'inline-block',
       marginInlineEnd: `${spacings.FULL}rem`,
       ...fontVariants.sansBold,
     }),
+  liveLabelCircle: ({ palette, spacings }: Theme) =>
+    css({
+      color: palette.LIVE_DARK,
+      borderRadius: '50%',
+      display: 'inline-block',
+      width: '.9375rem',
+      height: '.9375rem',
+      background:
+        'radial-gradient(circle,transparent 53%,hsla(180, 100%, 20%,.5) 53.5%,#006666 54%)',
```

Comment:
Let's use `palette.LIVE_DARK` here instead of hardcoding the colour:

```suggestion
        `radial-gradient(circle,transparent 53%,hsla(180, 100%, 20%,.5) 53.5%, ${palette.LIVE_DARK} 54%)`,
```